src/components/timeline/TrackNetwork.js

```diff
@@ -43,6 +45,11 @@ type CanvasProps = {|
   +rangeEnd: Milliseconds,
   +width: CssPixels,
   +networkTiming: MarkerTiming[],
+  +onHoveredMarkerChange: (
+    hoveredMarkerIndex: MarkerIndex | null,
+    mouseX?: CssPixels,
+    mouseY?: CssPixels
+  ) => mixed,
```

Comment:
`mixed` is the Flow way of saying "we don't care about what is returned". Indeed we don't use the return value, so it can return anything!
If we use `void` instead, then we _force_ the caller to pass a function that returns nothing. But in reality we don't need to force that :-)

Of course this discussion is purely theoretical because in this case the passed function doesn't return anything, but I believe `mixed` is in theory more correct.

Reply:
I know about the `mixed`, but since we don't use the return value at all and we don't also return anything, I don't see why we allow callers to return anything. It's like marking a `void` function mixed just in case we return something in the future, isn't it? But anyway, I don't feel strongly about it, it's fine by me either way.

Reply:
> It's like marking a void function mixed just in case we return something in the future, isn't it?

It's different.
When you implement a function, you know it returns something of a specific type.
Here we're reasoning about APIs. We have 2 components that interact with each other. Component `TrackNetworkCanvas` takes a callback, component `TrackNetwork` gives a callback. If we think of each of them separately, then `TrackNetworkCanvas` can accept callbacks that return nothing but also callbacks that return something -- as we don't use the return value. That's just as valid.

In this specific case, they're in the same file, and the Canvas one isn't even exported.
So here your argument works because they're meant to work one with the other, it's easy to change the second one if the first one changes too, etc.

So in short, I think both makes sense, just `any` is banned of course :-) I'll change to `void` because we're in the same file.

BTW: I had a look at what TypeScript does. In TypeScript, `undefined` and `void` are different values (`undefined` is the _value_ undefined, but `void` is the return value of a function that doesn't return anything). `unknown` is Flow's `mixed` but it's not needed in TS in this case. Indeed in TS when we type the callback with `=> void`, this also works with functions that return something.